openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -152,7 +152,11 @@ def process(self, instance):
                         json.dumps(preset, indent=4, sort_keys=True)
                     )
                 )
-
+            if (
+                preset["viewport_options"].get("reloadTextures")
+                and preset["viewport_options"].get("textures")
+            ):
+                lib.reload_textures()
             path = capture.capture(**preset)
```

Comment:
This should also pop `reloadTextures`?

```suggestion
            preset.pop("reloadTextures")  # not supported by `capture`
            path = capture.capture(**preset)
```